index.html

```diff
@@ -5,14 +5,11 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
-    
+
   </head>
   <body>
     <div id="app"></div>
-   
-    
-    
 
- <script type="module" src="/main.js"></script>
+    <script type="module" src="/main.js"></script>
```

Comment:
You should make this file disappear from this PR. In order to do that make sure that it's content is exactly the same as it was before. A quick and easy way would be changing branch (locally) to `dev`, copy whole content of `index.html`, change branch again to `first-btn` and paste it back. Then ofc make a commit and send it to the repo.